op-devstack/dsl/l2_cl.go

```diff
@@ -44,3 +53,72 @@ func (cl *L2CLNode) Start() {
 func (cl *L2CLNode) Stop() {
 	cl.control.L2CLNodeState(cl.inner.ID(), stack.Stop)
 }
+
+func (cl *L2CLNode) SyncStatus() *eth.SyncStatus {
+	ctx, cancel := context.WithTimeout(cl.ctx, DefaultTimeout)
+	defer cancel()
+	syncStatus, err := cl.inner.RollupAPI().SyncStatus(ctx)
+	cl.require.NoError(err)
+	return syncStatus
+}
+
+func (cl *L2CLNode) HeadBlockRef(label string) eth.BlockRef {
+	syncStatus := cl.SyncStatus()
+	targetWrap := reflect.ValueOf(syncStatus).Elem().FieldByName(label)
+	cl.require.True(targetWrap.IsValid(), "invalid label")
```

Comment:
Same here, like the supervisor status method suggestion, but on the op-node sync status type.